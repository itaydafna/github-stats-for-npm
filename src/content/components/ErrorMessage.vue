<template>
  <div id="container">
    <span>* Failed to fetch stats</span>
  </div>
</template>

<script>
export default {
  data() {
    return {};
  },
};
</script>

<style scoped>
#container {
  position: relative;
}
span {
  position: absolute;
  top: -15px;
  font-size: 0.8rem;
  color: #e42c31;
}
</style>
