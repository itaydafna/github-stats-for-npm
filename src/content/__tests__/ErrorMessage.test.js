import { mount } from '@vue/test-utils';
import ErrorMessage from '../components/ErrorMessage';

describe('ErrorMessage', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = mount(ErrorMessage);
  });
  test('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy();
  });
  it('renders an error message', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
