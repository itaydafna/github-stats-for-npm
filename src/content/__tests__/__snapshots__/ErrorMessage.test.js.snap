// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage renders an error message 1`] = `
<div
  id="container"
>
  <span>
    * Failed to fetch stats
  </span>
</div>
`;
